DESCRIPTION

```diff
@@ -36,6 +36,7 @@ Imports:
     stats,
     stringi (>= 1.6)
 Suggests:
+    random.cdisc.data (>= 0.3.16),
```

Comment:
what we could do here is instead of using the random.cdisc.data, just use the example data from rtables

Reply:
you can find out more on how i do this in autoslider and autoslider.core, just reexport the dataset

Reply:
It is a depends, no need to reexport